packages/eas-cli/src/credentials/ios/actions/BuildCredentialsUtils.ts

```diff
@@ -1,13 +1,16 @@
+import { Platform } from '@expo/eas-build-job';
 import nullthrows from 'nullthrows';
 
+import { getBundleIdentifier } from '../../../build/ios/bundleIdentifier';
+import { resolveWorkflow } from '../../../build/utils/workflow';
```

Comment:
code outside `build` directory should not be aware of where bundleIdentifier is located, this resolve function should be part of the getBundleIdentifier

Also, I think the function for accessing bundleIdentifier should not be in the `build` directory